bin/build_in_docker.sh

```diff
@@ -19,9 +22,16 @@ remove_image() {
 }
 
 main() {
+  cp ./bin/docker/Dockerfile . &&
   build_image &&
-    docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -e "ARGS=$*" tmp:build &&
+    docker run \
+      --rm \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      -v $MAVEN_HOME_LOCAL/repository:$MAVEN_HOME_DOCKER/repository \
+      -e "ARGS=$*" \
+      tmp:build &&
     remove_image
+  rm Dockerfile 2>/dev/null
```

Comment:
I understand that you're copying file from `./bin/docker/Dockerfile` into the current directory `.` for building the image tagged as `tmp:build`.

Can't we build the image with `-f` parameter in order to avoid having to copy Dockerfile then remove it?

Something like this in the `buil_image` function: 
```bash
docker build . --rm -f ./bin/docker/Dockerfile -t tmp:build
``` 

which will lead to this in `main` function:
```bash
main() {
  build_image &&
    docker run \
      --rm \
      -v /var/run/docker.sock:/var/run/docker.sock \
      -v $MAVEN_HOME_LOCAL/repository:$MAVEN_HOME_DOCKER/repository \
      -e "ARGS=$*" \
      tmp:build &&
    remove_image
}
```

Reply:
As mentioned in another comment on this pull request, I'll keep the build-related enhancement suggestions in the backlog for now to prioritise more pressing matters.